astro/src/content/docs/get-started/download-and-install/system-requirements.mdx

```diff
@@ -125,12 +125,19 @@ In most cases, if you have thousands of users and entities, `10GB` of disk space
 
 ## Network Access
 
-FusionAuth downloads the MySQL JDBC driver at install time. Due to the drivers' licensing, it cannot be bundled into the application. Prior to 1.16.0, the drivers were downloaded no matter which database you used. After that version, they are only downloaded if you are using MySQL.
+FusionAuth offers instructions on downloading the MySQL JDBC driver at install time. Due to the drivers' licensing, it cannot be bundled into the application. Prior to 1.16.0, the drivers were downloaded no matter which database you used. Prior to 1.40.0, the drivers were downloaded if you are using MySQL.
 
 If you are running FusionAuth in an environment with no network access and are using MySQL, you must download the MySQL driver jar file and place it in `fusionauth-app/web/WEB-INF/lib`.
 
 ### Paid Editions
 
-If you are using a paid edition of FusionAuth, the instance must have outbound network connectivity for license id verification and metrics reporting. You must also allow access to enable advanced features provided by FusionAuth Reactor cloud services.
+If you are using the community plan, you can block all outbound network connectivity. Lack of connectivity will not affect the product. 
+
+Once on a paid plan, you must enable network access to FusionAuth or your license will not function properly. You must also allow access to enable advanced features provided by FusionAuth Reactor cloud services.
```

Comment:
The "you must enable" and "you must also" sounded too demanding, but it's up to you if you want to change it.
```suggestion
Once on a paid plan, you must enable network access to FusionAuth, or your license will not function properly. Advanced features provided by FusionAuth Reactor cloud services require additional outbound network access.
```